docs/UserGuide.md

```diff
@@ -228,19 +228,25 @@ Use this command to edit his/her details easily!
 
 **Syntax:**
 
-`edit INDEX [{SPECIFIER}/{DATA}]`
+`edit INDEX {SPECIFIER}/{DATA}...`
 
 _Here are some important requirements for you to take note:_
 
 * `INDEX` refers to the index of the contact you wish to edit in the current displayed list.
   * `INDEX` must be a **positive integer**.
 * At least one field must be provided.
-  * Note that when editing tags/mods taken, the existing tags/mods taken of/by the person will be removed
-
-  _[i.e  adding of tags and mods taken are not cumulative]_
-  * If you wish to remove all tags/mods from the person, simply type `t/` / `mt/`.
-* We plan to make a new function in the future to make both tags and mods taken cumulative, stay tuned!
 
+For the following fields, they are considered a `SuperField`.
+* Modules
+* Tags
+
+A `SuperField`
+can contain many inputs in that single field. When using edit, the command checks
+for if that input is already in that `SuperField`, if it is, it will remove it.
+If not, then it will add the input into the `SuperField`.
```

Comment:
```suggestion
A `SuperField` can contain many inputs in that single field.
When using edit, the command looks for each input in the `SuperField`:
* If the input already exists in the `SuperField` it will be removed.
* Otherwise, the input will be added into the `SuperField`.
```

Also can consider putting `SuperField` in the DG's glossary.